src/model/differ.js

```diff
@@ -238,9 +232,7 @@ export default class Differ {
 
 				// If the wrap took the element from the graveyard, mark that the element from the graveyard was removed.
```

Comment:
Comment need to be updated.

Reply:
Mmmm... I don't understand why?

Reply:
As I mentioned belove, we should remove all the code related to wrap and unwrap operations which is not used anymore.